packages/wallets/src/i18n.ts

```diff
@@ -0,0 +1,61 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+const DEFAULT_LANGUAGE = 'EN';
+
+export const getInitialLanguage = () => {
+    const URLParams = new URLSearchParams(window.location.search);
+    const queryLang = URLParams.get('lang');
+    const localStorageLanguage = localStorage.getItem('i18n_language');
+
+    if (queryLang) {
+        localStorage.setItem('i18n_language', queryLang.toUpperCase());
+        return queryLang.toUpperCase();
+    }
+
+    if (localStorageLanguage) {
+        return localStorageLanguage;
+    }
+
+    return DEFAULT_LANGUAGE;
+};
+
+let currentLanguage = i18n.language ?? getInitialLanguage();
+
+export const setLanguage = (lang: string) => {
+    currentLanguage = lang || DEFAULT_LANGUAGE;
+    i18n.changeLanguage(lang);
+};
+
+export const getCurrentLanguage = () => currentLanguage;
+
+const resources = {
+    EN: {
+        translations: {
+            Deposit: 'Deposit',
+            'Reset balance': 'Reset balance',
+            Transactions: 'Transactions',
+            Transfer: 'Transfer',
+            Withdraw: 'Withdraw',
+        },
+    },
+    FR: {
+        translations: {
+            Deposit: 'Dépôt',
+            'Reset balance': 'Réinitialiser le solde',
+            Transactions: 'Transactions',
+            Transfer: 'Transfert',
+            Withdraw: 'Retirer',
+        },
+    },
```

Comment:
```suggestion
    EN: {
        translations: {
            Deposit: 'Deposit',
            'Reset balance': 'Reset balance',
            Transactions: 'Transactions',
            Transfer: 'Transfer',
            Withdraw: 'Withdraw',
        },
    },
    FR: {
        translations: {
            Deposit: 'Dépôt',
            'Reset balance': 'Réinitialiser le solde',
            Transactions: 'Transactions',
            Transfer: 'Transfert',
            Withdraw: 'Retirer',
        },
    },
```
It works fine in this case because you already have the translations updated here (which is usually done after extraction -> translation update by content -> download)